datastore/blockstore.go

```diff
@@ -64,7 +64,7 @@ func (bs *bstore) HashOnRead(enabled bool) {
 // Get returns a block from the blockstore.
 func (bs *bstore) Get(ctx context.Context, k cid.Cid) (blocks.Block, error) {
 	if !k.Defined() {
-		log.Error(ctx, "Undefined CID in blockstore")
+		log.ErrorContext(ctx, "Undefined CID in blockstore")
```

Comment:
suggestion: We should remove the error logging here since an error is returned to the caller.